extensions/wikia/Search/classes/Services/CombinedSearchService.php

```diff
@@ -132,10 +149,22 @@ public function phraseSearchForArticles( $query, $namespaces, $langs, $hubs = nu
 	 * @param $maxArticlesPerWiki
 	 * @param $wikiId
 	 * @param $wikiLang
+	 *
 	 * @return array
 	 */
 	protected function querySolrForArticles( $query, $namespaces, $maxArticlesPerWiki, $wikiId, $wikiLang ) {
-		$requestedFields = [ "title", "url", "id", "score", "pageid", "lang", "wid", "article_quality_i", "article_type_s", Utilities::field( 'html', $wikiLang ) ];
+		$requestedFields = [
+			"title",
+			"url",
+			"id",
+			"score",
+			"pageid",
+			"lang",
+			"wid",
+			"article_quality_i",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "article_quality_i" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)